next/components/ui/AccountSuccessAlert/AccountSuccessAlert.tsx

```diff
@@ -0,0 +1,33 @@
+import ArrowRightIcon from '@assets/images/forms/arrow-right.svg'
```

Comment:
would like to separate these sections from the UI of bratislava.sk - this is mainly because much of what presently resides within components/ui will need refactoring and I want to separate newly created components (which use slightly different design) from this cohort

we're putting all the components using this 'new' design into `forms` - this directory might get a rename to be more fitting (let's do that elsewhere), as it will contain also sections from city account, but for now let's place everything there

while on this topic - please find a reasonable subdirectory for this within `/forms`, perhaps `/forms/segments` ? would call it a `section`, but section already has a connotation within our CMS :) .. naming things is hard

you can do this in a followup PR and do it with the 'sections' (or components) from the login PR as well if there are any that currently resides within ui, I likely missed this in that PR